uamqp/receiver.py

```diff
@@ -74,9 +74,13 @@ def _incoming_transfer(self, frame):
         if not self.received_delivery_id and not self._received_payload:
             pass  # TODO: delivery error
         if self._received_payload or frame[5]:  # more
-            raise NotImplementedError()  # TODO
+            self._received_payload += bytes(frame[11])  # TODO, is it possible to use memoryview directly for concatenation?
         if not frame[5]:
-            message = decode_payload(frame[11])
+            if self._received_payload:
+                message = decode_payload(memoryview(self._received_payload))  # TODO, is it possible to use memoryview directly for concatenation?
+                self._received_payload = b""
```

Comment:
Are we going to add some tests around this?

Reply:
I have added a test case to receive large and small messages, it runs with no problem